pnpm-lock.yaml

```diff
@@ -585,6 +587,17 @@ packages:
       '@babel/helper-explode-assignable-expression': 7.16.0
       '@babel/types': 7.16.0
 
+  /@babel/helper-compilation-targets/7.16.3:
+    resolution: {integrity: sha512-vKsoSQAyBmxS35JUOOt+07cLc6Nk/2ljLIHwmq2/NM6hdioUaqEXq/S+nXvbvXbZkNDlWOymPanJGOc4CBjSJA==}
+    engines: {node: '>=6.9.0'}
+    peerDependencies:
+      '@babel/core': ^7.0.0
+    dependencies:
+      '@babel/compat-data': 7.16.4
+      '@babel/helper-validator-option': 7.14.5
+      browserslist: 4.18.1
+      semver: 6.3.0
+
```

Comment:
There are a lot of changes in the lockfile that seem unnecessary, is that just due to the different lockfileVersion?

Reply:
I think it's the result of installing `@rollup/pluginutils`

Reply:
This project hasn't been updated to pnpm 7 yet. That could be the issue?

Reply:
Oooh yup, that seemed to be it. Reinstalled with v6